src/EnergyPlus/ChilledCeilingPanelSimple.hh

```diff
@@ -0,0 +1,347 @@
+	extern int const CondCtrlVariedOff; // Condensation control--variable off, system modulates to keep running if possible
+
+	// DERIVED TYPE DEFINITIONS
+
+	//MODULE VARIABLE DECLARATIONS:
+	extern int NumCoolingPanels;
+	extern Array1D< Real64 > CoolingPanelSource; // Need to keep the last value in case we are still iterating
+	extern Array1D< Real64 > CoolingPanelSrcAvg; // Need to keep the last value in case we are still iterating
+	extern Array1D< Real64 > ZeroSourceSumHATsurf; // Equal to the SumHATsurf for all the walls in a zone with no source
+
+	// Record keeping variables used to calculate CoolingPanelSrcAvg locally
+	extern Array1D< Real64 > LastCoolingPanelSrc; // Need to keep the last value in case we are still iterating
+	extern Array1D< Real64 > LastSysTimeElapsed; // Need to keep the last value in case we are still iterating
+	extern Array1D< Real64 > LastTimeStepSys; // Need to keep the last value in case we are still iterating
+	extern Array1D_bool CheckEquipName;
+	extern Array1D_bool SetLoopIndexFlag; // get loop number flag
+
+	//SUBROUTINE SPECIFICATIONS FOR MODULE Simple Chilled Ceiling Panel
+
+	// Types
+
+	struct CoolingPanelParams
+	{
+		// Members
+		std::string EquipID;
+		int EquipType;
+		std::string Schedule;
+		Array1D_string SurfaceName;
+		Array1D_int SurfacePtr;
+		int ZonePtr;
+		int SchedPtr;
+		int WaterInletNode;
+		int WaterOutletNode;
+		int TotSurfToDistrib;
+		int ControlCompTypeNum;
+		int CompErrIndex;
+		int ControlType;
+		std::string ColdSetptSched;
+		int ColdSetptSchedPtr;
+		int CondCtrlType;
+		Real64 CondDewPtDeltaT;
+		int CondErrIndex;
+		Real64 ColdThrottlRange;
+		Real64 RatedWaterTemp;
+		Real64 RatedCapacity;
+		Real64 UA;
+		Real64 Offset;
+		Real64 WaterMassFlowRate;
+		Real64 WaterMassFlowRateMax;
+		Real64 RatedWaterFlowRate;
+		Real64 WaterVolFlowRateMax;
+		Real64 WaterInletTempStd;
+		Real64 WaterInletTemp;
+		Real64 WaterInletEnthalpy;
+		Real64 WaterOutletTempStd;
+		Real64 WaterOutletTemp;
+		Real64 WaterOutletEnthalpy;
+		Real64 RatedZoneAirTemp;
+		Real64 FracRadiant;
+		Real64 FracConvect;
+		Real64 FracDistribPerson;
+		Array1D< Real64 > FracDistribToSurf;
+		Real64 TotPower;
+		Real64 Power;
+		Real64 ConvPower;
+		Real64 RadPower;
+		Real64 TotEnergy;
+		Real64 Energy;
+		Real64 ConvEnergy;
+		Real64 RadEnergy;
+		int LoopNum; // plant loop index
+		int LoopSideNum; // plant loop side index
+		int BranchNum; // plant loop branch index
+		int CompNum; // plant loop component index
+		int CoolingPanelLoadReSimIndex;
+		int CoolingPanelMassFlowReSimIndex;
+		int CoolingPanelInletTempFlowReSimIndex;
+
+		// Default Constructor
+		CoolingPanelParams() :
+			EquipType( 0 ),
+			ZonePtr( 0 ),
+			SchedPtr( 0 ),
+			WaterInletNode( 0 ),
+			WaterOutletNode( 0 ),
+			TotSurfToDistrib( 0 ),
+			ControlCompTypeNum( 0 ),
+			CompErrIndex( 0 ),
+			ControlType( 0 ),
+			ColdSetptSchedPtr( 0 ),
+			CondCtrlType( 0 ),
+		    CondDewPtDeltaT( 0.0 ),
```

Comment:
@Myoldmopar Weird...didn't see this in Xcode, but I deleted what was there and repositioned it again so that EquipType lines up with CondDewPtDeltaT.  Hopefully that resolves the problem?